CODEOWNERS

```diff
@@ -6,15 +6,15 @@
 # To be explicit: we will never accept changes to this file adding people from outside the Homebrew GitHub
 # organisation. If you are not a Homebrew maintainer: you do not personally "own" or "maintain" any formulae.
 
-CODEOWNERS            @MikeMcQuaid @Homebrew/plc @Homebrew/tsc
-CONTRIBUTING.md       @MikeMcQuaid @Homebrew/tsc
-tap_migrations.json   @MikeMcQuaid @Homebrew/tsc
-.github/              @MikeMcQuaid @Homebrew/tsc
-cmd/                  @MikeMcQuaid
+CODEOWNERS                        @Homebrew/tsc @Homebrew/plc @MikeMcQuaid 
```

Comment:
`make sure the team @Homebrew/plc exists, is publicly visible, and has write access to this repository`

Reply:
It doesn't and I don't think it should

Reply:
can't be in `CODEOWNERS` then